lua/lvim/config/settings.lua

```diff
@@ -51,6 +51,23 @@ M.load_options = function()
 
   vim.opt.shortmess:append "c"
 
+  -- Defaults:
+  --   Neovim: !,'100,<50,s10,h
+  --
+  -- - ! save/restore global variables (only all-uppercase variables)
+  -- - '100 save/restore marks from last 100 files
+  -- - <50 save/restore 50 lines from each register
+  -- - s10 max item size 10KB
+  -- - h do not save/restore 'hlsearch' setting
```

Comment:
I actually think that these two defaults are good.
Restoring the highlights of search on a reload probably loses context.

For reference `:help sd`

Reply:
I thought it was the other way around with `h` command. It's a bit confusing how they named these variables 🥲